index.html

```diff
@@ -1,160 +1,278 @@
-                </div>
-                
-                <ul class="about_flex">
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/JS_icon.png" alt="">
-                        <h4 class="about_lang">JavaScript</h4>
-                    </li>
-    
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/html_icon.png" alt="">
-                        <h4 class="about_lang">HTML</h4>
-                    </li>
-    
-                    <li class="about_skill">
-                        <img class="about_img" src="./image/css_icon.png" alt="">
-                        <h4 class="about_lang">CSS</h4>
-                    </li>
-    
-                    
-                </ul>
+              <a href="#" class="about_icon"
+                ><img src="./image/github_icon.png" alt="github"
+              /></a>
             </li>
-            
-
-            <li class="lang_tech border">
-                <h3 class="lang_title">Frameworks</h3>
-                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/Linkedin icon.png" alt="linkedin_icon"
+              /></a>
             </li>
-
-            <li class="lang_tech border">
-                <h3 class="lang_title">Skills</h3>
-                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/V_icon.png" alt="V_icon"
+              /></a>
             </li>
-
-
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/twitter_icon.png" alt="twitter_icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/m_icon.png" alt="m_icon"
+              /></a>
+            </li>
+          </ul>
+    
+          <a href="#" class="button2">Get my resume</a>
+        </div>       
+        <ul class="about_group">
+          <li>
+            <div class="lang_tech">
+              <h3 class="lang_title">Languages</h3>
+              <img class="lang_arrow" src="./image/Disabled.png" alt="" />
+            </div>
+  
+            <ul class="about_flex">
+              <li class="about_skill">
+                <img class="about_img" src="./image/JS_icon.png" alt="" />
+                <h4 class="about_lang">JavaScript</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/html_icon.png" alt="" />
+                <h4 class="about_lang">HTML</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/css_icon.png" alt="" />
+                <h4 class="about_lang">CSS</h4>
+              </li>
+            </ul>
+          </li>
```

Comment:
- [ ] languages icons disappear in the desktop version please it would be better if you fix that in order to match the design and they should be aligned horizontally
-Yours 
<img width="535" alt="Screen Shot 2022-10-01 at 22 15 43" src="https://user-images.githubusercontent.com/61790587/193428914-73bd9fc1-1fa1-45bc-ad6a-4fa0c6938a05.png">
Figma
<img width="337" alt="Screen Shot 2022-10-01 at 22 27 17" src="https://user-images.githubusercontent.com/61790587/193428955-db4c24b8-1fe9-4697-8647-b569eadc00df.png">